data/explorations/all_interactions/all_interactions.yaml

```diff
@@ -76,6 +76,44 @@ states:
       id: Continue
       solution: null
     param_changes: []
+  Drag And Drop Sort:
```

Comment:
This won't work without enabling interaction in feconf. AFAIK this won't solve the exception problem in frontend.